### PR TITLE
feat: secondary constructors in inline classes

### PR DESCRIPTION
**Pull request by danil-pavlov**

### CODE REVIEW

File: docs/topics/inline-classes.md [hunk 1]
```diff
@@ -40,29 +36,39 @@ usages (similar to how content of [inline functions](inline-functions.md) is inl
 ## Members
 
 Inline classes support some functionality of regular classes. In particular, they are allowed to declare properties and 
-functions, and have the `init` block:
+functions, have the `init` block and [secondary constructors](classes.md#secondary-constructors):
 
 ```kotlin
 @JvmInline
-value class Name(val s: String) {
+value class Person(private val firstName: String) {
```
Comment (zarechenskiy) [inline docs/topics/inline-classes.md:43]: Probably, `firstName` -> `fullName`

**Review by zarechenskiy — APPROVED** [review]

File: docs/topics/inline-classes.md [hunk 2]
```diff
@@ -21,10 +21,6 @@ To declare an inline class for the JVM backend, use the `value` modifier along w
 value class Password(private val s: String)
 ```
 
-> The `inline` modifier for inline classes is deprecated.
-> 
-{type="warning"}
-
 An inline class must have a single property initialized in the primary constructor. At runtime, instances of the inline
```
Comment (zarechenskiy) [inline docs/topics/inline-classes.md:24]: Also, it looks like we have to call them "inline values class"

File: docs/topics/inline-classes.md [hunk 3]
```diff
@@ -40,29 +36,39 @@ usages (similar to how content of [inline functions](inline-functions.md) is inl
 ## Members
 
 Inline classes support some functionality of regular classes. In particular, they are allowed to declare properties and 
-functions, and have the `init` block:
+functions, have the `init` block and [secondary constructors](classes.md#secondary-constructors):
```
Comment (sarahhaggarty) [inline docs/topics/inline-classes.md:39]:
```suggestion
functions, have an `init` block and [secondary constructors](classes.md#secondary-constructors):
```

**Review by sarahhaggarty — APPROVED** [review]
Looks great! Just one tiny suggestion.

**Pull request merged**